book/iteration.tex

```diff
@@ -554,7 +554,7 @@ \section{Control Flow Statements ({\tt last}, {\tt next}, etc.)}
 print('Done!')
 \end{verbatim}
 %
-Here, the loop terminates if the user types ``done.'' If not, the user input is checked by the {\tt is-valid} subroutine; if the subroutine returns a true value, the processing continues forward; if it returns a false value, then the control flow starts again at the beginning of the body of the loop, so the user is prompted again to submit a valid input.
+Here, the loop terminates if the user types ``done''. If not, the user input is checked by the {\tt is-valid} subroutine; if the subroutine returns a true value, the processing continues forward; if it returns a false value, then the control flow starts again at the beginning of the body of the loop, so the user is prompted again to submit a valid input.
```

Comment:
This correction is wrong. Although this is not logical (at least in my view), English typography always puts the closing quote marks after the punctuation symbol, so that QUOTE ... if the user types ``done.'' UNQUOTE is the correct way of doing it in English.

Reply:
You're totally right about this. There seems to be different styles regarding quotations and puntuaction. In this case, I thought style could be sacrificed for the sake of comprehension but this might be me nitpicking. Not really an issue at all.